### PR TITLE
feat: Kotlin 2.0.20-Beta1 EAP release details

### PR DESCRIPTION
**Pull request by koshachy**

### CODE REVIEW

File: docs/topics/eap.md [hunk 1]
```diff
@@ -38,4 +38,21 @@ check [our instructions on how to configure your build to support this version](
...
-_No preview versions are currently available._
+<!-- _No preview versions are currently available._ -->
+
+<table>
+    <tr>
+        <th>Build info</th>
+        <th>Build highlights</th>
+    </tr>
+    <tr>
+        <td><strong>2.0.0-Beta1</strong>
```
Comment (sarahhaggarty) [inline docs/topics/eap.md:49]:
```suggestion
        <td><strong>2.0.20-Beta1</strong>
```

File: docs/topics/eap.md [hunk 2]
```diff
@@ -38,4 +38,21 @@ check [our instructions on how to configure your build to support this version](
...
+            <p>Released: <strong>June 18, 2024</strong></p>
+            <p><a href="https://github.com/JetBrains/kotlin/releases/tag/v2.0.20-Beta1" target="_blank">Release on GitHub</a></p>
+        </td>
+        <td>
+            <p>A tooling release for Kotlin 2.0.0</p>
```
Comment (sarahhaggarty) [inline docs/topics/eap.md:54]:
```suggestion
            <p>A tooling release for Kotlin 2.0.20</p>
```

File: docs/topics/eap.md [hunk 3]
```diff
@@ -38,4 +38,21 @@ check [our instructions on how to configure your build to support this version](
...
+            <p>For more details, please refer to the <a href="https://github.com/JetBrains/kotlin/releases/tag/v2.0.20-Beta1">changelog</a> or <a href="whatsnew-eap.md">What's new in Kotlin 2.0.0-Beta1</a>.</p>
```
Comment (sarahhaggarty) [inline docs/topics/eap.md:55]:
```suggestion
            <p>For more details, please refer to the <a href="https://github.com/JetBrains/kotlin/releases/tag/v2.0.20-Beta1">changelog</a> or <a href="whatsnew-eap.md">What's new in Kotlin 2.0.20-Beta1</a>.</p>
```

File: docs/topics/eap.md (hunk 2 above)
Comment (koshachy) [reply docs/topics/eap.md:54]: 2.0.20 is not out yet.
all x.x.20 releases a tooling releases for the previous ones.

File: docs/topics/whatsnew-eap.md [hunk 4]
```diff
@@ -9,1615 +9,90 @@ _[Released: %kotlinEapReleaseDate%](eap.md#build-details)_
...
+Note that even with this annotation, the compiler still reports warnings when the `.copy()` function is called.
 
-### Generate lambda functions using invokedynamic
+If you want to opt in to the new behavior already in %kotlinEapVersion% for a whole module rather than in individual classes,
+you can use the `-Xconsistent-data-class-copy-visibility` compiler option.
+This option has the same effect as adding the '@ConsistentCopyVisibility' annotation to all data classes in a module.
 
-Kotlin %kotlinEapVersion% introduces a new default method for generating lambda functions using `invokedynamic`.
-This change reduces the binary sizes of applications compared to the traditional anonymous class generation.
+## Kotlin/Native: support for bitcode embedding removed
```
Comment (danil-pavlov) [inline docs/topics/whatsnew-eap.md:68]:
```suggestion
## Kotlin/Native: Support for bitcode embedding removed
```

Comment (danil-pavlov) [reply docs/topics/whatsnew-eap.md:68]: Not sure if a capital letter should follow the colon, but for consistency with other headers

Comment (koshachy) [reply docs/topics/whatsnew-eap.md:68]: Thank you, updated the PR

**Pull request merged**